app/components/content/content.js

```diff
@@ -321,8 +321,20 @@ const Content = ({ p2p, content, renderRow }) => {
               isLoading={isDeleting}
               onClick={async () => {
                 setIsDeleting(true)
-                await p2p.delete(content.rawJSON.url)
-                history.push('/')
+                const deleteFiles = true
+                const { response } = await remote.dialog.showMessageBox(
+                  remote.getCurrentWindow(),
+                  {
+                    type: 'warning',
+                    buttons: ['Delete all files', 'Cancel'],
+                    message: 'Do you really wanna?'
+                  }
+                )
+
+                if (response === 0) {
```

Comment:
In order to follow existing patterns in other components

```suggestion
                if (response === 1) return

```

Reply:
Didn't know this trick :) Thanks!